db/R/dbfiles.R

```diff
@@ -147,18 +147,17 @@ dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate,
 ##' @param con database connection object
 ##' @param hostname the name of the host where the file is stored, this will default to the name of the current machine
 ##' @param params database connection information
-##' @param exact.dates setting to include start and end date in input query
```

Comment:
ignore.dates was part of @tonygardella's PR. This again looks like something that's reverting to before his updates.  @jam2767 given all the problems you're having and the relatively tiny bits of code that you're adding, you might just want to start with a fresh clone of PEcAn, create a branch specific for this work, re-add your bits, and then push it to your github.